### ISSUE TITLE
Getting "got an unexpected keyword argument 'type_'" with basic example

### ISSUE THREAD

**Issue by lohmander** [strawberry-graphql/strawberry-graphql-django#46]
I've been banging my head against this for a little while and compared to the examples on here, but haven't been able to figure this one out.

My model

```python
from django.db import models


class Contact(models.Model):
    first_name = models.CharField(max_length=100)
    last_name = models.CharField(max_length=100)
    email = models.EmailField()
```

Strawberry types

```python
import strawberry
from strawberry.django import auto
from core import models


@strawberry.django.type(models.Contact)
class Contact:
    id: auto
```

Error

```sh
  File "Projects/project/project/urls.py", line 19, in <module>
    from core.schema import schema
  File "Projects/project/core/schema.py", line 3, in <module>
    from core.types import Contact
  File "Projects/project/core/types.py", line 7, in <module>
    class Contact:
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/type.py", line 154, in wrapper
    return process_type(cls, model, filters=filters, **kwargs)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/type.py", line 131, in process_type
    fields = get_fields(django_type)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/type.py", line 89, in get_fields
    field = get_field(django_type, field_name, field_annotation)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/type.py", line 29, in get_field
    field = StrawberryDjangoField(
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/fields/field.py", line 60, in __init__
    super().__init__(graphql_name=graphql_name, python_name=python_name, **kwargs)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/ordering.py", line 56, in __init__
    super().__init__(**kwargs)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/filters.py", line 138, in __init__
    super().__init__(**kwargs)
  File "Projects/project/env/lib/python3.9/site-packages/strawberry_django/pagination.py", line 24, in __init__
    super().__init__(**kwargs)
TypeError: __init__() got an unexpected keyword argument 'type_'
```

Any ideas where I'm going wrong, or is this a bug?

Thanks

**Comment by patrick91** [comment]
@lohmander can show me what version of strawberry are you using?

we just release a big internal update and it might have broken strawberry django

**Comment by lohmander** [comment]
Sure thing:

strawberry-graphql==0.69.2
strawberry-graphql-django==0.2.3

**Comment by patrick91** [comment]
@lohmander can you try to install `strawberry-graphql<0.69` ? 

we'll ship a fix for the latest version soon 😊

**Comment by lohmander** [comment]
@patrick91 this did indeed fix the issue 👍 thanks!

**Comment by la4de** [comment]
Temporal solution #47 and relesed. Please update to v0.2.4. Proper fix (#48) will be done later.
